documentation/getting-started.adoc

```diff
@@ -0,0 +1,242 @@
+
+For OpenShift, Kafka broker is provided in the form of a template. The cluster can be deployed from the template either
+using command line or using the OpenShift console. To create the ephemeral cluster, following command should be executed:
+
+[source]
+oc new-app strimzi-ephemeral
+
+Similarly to deploy persistent KAfka cluster, following command should be run:
+
+[source]
+oc new-app strimzi-persistent
+
+=== Kafka Connect
+
+The Cluster Controller can also deploy a https://kafka.apache.org/documentation/#connect[Kafka Connect] clusters which
+can be used with either of the Kafka broker deployments described above. It is implemented as a Deployment with a
+configurable number of workers. The default image currently contains only the Connectors distributed with Apache Kafka
+Connect -`FileStreamSinkConnector` and `FileStreamSourceConnector`. The REST interface for managing the Kafka Connect
+cluster is exposed internally within the Kubernetes/OpenShift cluster as service `kafka-connect` on port `8083`.
+
+Example ConfigMaps and the details about the ConfigMap format for deploying Kafka Connect can be found in
+<<kafka_connect_config_map_details>>.
+
+==== Deploying to Kubernetes
+
+To deploy Kafka Connect on Kubernetes, the corresponding ConfigMap has to be created. An example ConfigMap can be
+created using following command:
+
+[source]
+kubectl apply -f resources/kubernetes/kafka-connect.yaml
+
+==== Deploying to OpenShift
+
+On OpenShift, Kafka Connect is provided in the form of a template. It can be deployed from the template either
+using command line or using the OpenShift console. To create Kafka Connect cluster from the command line, following
+command should be run:
+
+[source]
+oc new-app strimzi-connect
+
+==== Using Kafka Connect with additional plugins
+
+Strimzi Docker images for Kafka Connect contain by default only the `FileStreamSinkConnector` and
+`FileStreamSourceConnector` connectors which are part of the Apache Kafka project.
+
+But they configure Kafka Connect to automatically load all plugins/connectors which are present in the
+`/opt/kafka/plugins` directory during startup. There are two ways how to add custom plugins into this directory:
+
+- Using custom Docker image
+- Using OpenShift build system and with the Strimzi S2I image
+
+===== Create a new image based on `strimzi/kafka-connect`
+
+Strimzi provides its own Docker image for running Kafka Connect which can be found on Docker Hub as
+https://hub.docker.com/r/strimzi/kafka-connect/[`strimzi/kafka-connect`]. This image could be used as a base image for
+building a new custom image with additional plugins. Following steps describe the process for creating the custom image:
+
+1. Create a new `Dockerfile` which uses `strimzi/kafka-connect` as base image
+
+[source,Dockerfile]
+FROM strimzi/kafka-connect:latest
+USER root:root
+COPY ./my-plugin/ /opt/kafka/plugins/
+USER kafka:kafka
+
+2. Build the Docker image and upload it to your Docker repository
```

Comment:
We've not really been using you/your etc in the docs so far, so maybe "upload it to the appropriate Docker repository"